tests/Opus/Doi/DoiManagerTest.php

```diff
@@ -61,6 +63,51 @@ public function testConstructorAlt()
         $this->assertNotNull($doiManager);
     }
 
+    public function testGetDoiLogger()
+    {
+        $logService = LogService::getInstance();
+        $logService->setConfig(new \Zend_Config([], true));
```

Comment:
There is a configuration for the tests. Isn't this code here replacing that with an empty configuration? What about the `workspacePath` for instance? How does this `LogService` object here find the folder for logging?